packages/alfa-dom/src/node/element.ts

```diff
@@ -372,7 +372,9 @@ export namespace Element {
           Attribute.fromAttribute(attribute).run()
         ),
         children,
-        Option.from(json.style).map(Block.from)
+        json.style?.length === 0
+          ? None
+          : Option.from(json.style).map(Block.from)
```

Comment:
Following what is done in `h.element`:
https://github.com/Siteimprove/alfa/blob/main/packages/alfa-dom/src/h.ts#L94